src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -813,10 +813,19 @@ bool DatabaseCatalog::isDatabaseExist(const String & database_name) const
     return databases.end() != databases.find(database_name);
 }
 
-Databases DatabaseCatalog::getDatabases() const
+Databases DatabaseCatalog::getDatabases(bool skip_datalake_catalogs) const
 {
     std::lock_guard lock{databases_mutex};
-    return databases;
+    if (skip_datalake_catalogs)
```

Comment:
So if `skip_datalake_catalogs==true`, we **do not skip**?

Reply:
Will fix name